apiserver/facades/client/application/application.go

```diff
@@ -59,9 +59,14 @@ var ClassifyDetachedStorage = storagecommon.ClassifyDetachedStorage
 
 var logger = loggo.GetLogger("juju.apiserver.application")
 
+// APIv18 provides the Application API facade for version 18.
```

Comment:
Maybe it would be a good moment to start commenting what is new in the version.

Reply:
There is a comment in register.go on what is added to this version. Could be added to the discussion on best practice for adding a new facade revision.